app/controllers/api/v1x2/portfolio_items_controller.rb

```diff
@@ -0,0 +1,19 @@
+module Api
+  module V1x2
+    class PortfolioItemsController < Api::V1x1::PortfolioItemsController
+      def show
+        portfolio_item = if params[:showDiscarded] == "true"
+                           model.with_discarded.discarded.find(params.require(:id))
```

Comment:
Does it finds only among discarded items or all items? The spec should test when the item exists and `:showDiscarded` is also true. The call will also return the normal item.

Reply:
Changed to find first in normal items, if not found, then try in discarded items.